targets/ramips-mt7621

```diff
@@ -53,6 +53,14 @@ device('d-link-dir-878-a1', 'dlink_dir-878-a1')
 device('d-link-dir-882-a1', 'dlink_dir-882-a1')
 
 
+-- Genexis
+
+device('genexis-inteno-pulse-ex400', 'genexis_pulse-ex400', {
```

Comment:
With https://github.com/openwrt/openwrt/commit/ea97c49766ad55cdfedcf7bd3eaf3b2a66d29730 this should probably be just the one name:

```suggestion
device('genexis-pulse-ex400', 'genexis_pulse-ex400', {
```